OpenIPSL/Examples/Machines/PSSE/GENROU.mo

```diff
@@ -23,37 +23,16 @@ model GENROU
     Xppq=0.2,
     R_a=0,
     Xpp=0.2,
-    H=4.28) annotation (Placement(transformation(extent={{-100,-20},{-60,20}})));
+    H=4.28) annotation (Placement(transformation(extent={{-100,-20},{
```

Comment:
Yes I wasn't aware what kind of rules there were. I used often the Dymola rearrange code whenever working on the annotations in text mode (which was quite often).